waku/v2/waku_archive/retention_policy/retention_policy_capacity.nim

```diff
@@ -8,8 +8,8 @@ import
   chronicles,
   chronos
 import
-  ../driver,
-  ../retention_policy
+  ../driver_base,
+  ../retention_policy_base
```

Comment:
Nit: I wouldn't call this `_base`. See my comment elsewhere on the established pattern here.